content/ch-gates/proving-universality.ipynb

```diff
@@ -17,28 +17,302 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2 Non-Clifford Gates <a id='non-clifford'></a>\n",
+    "\n",
+    "The Clifford gates are very important, but they are not powerful on their own. In order to do any quantum computation, we need gates that are not Cliffords. Three important examples are arbitrary rotations around the three axes of the qubit, $R_x(\\theta)$, $R_y(\\theta)$ and $R_z(\\theta)$.\n",
+    "\n",
+    "Let's focus on $R_x(\\theta)$. As we saw above, any unitary can be expressed in an exponential form using a Hermitian matrix. For this gate, we find\n",
+    "\n",
+    "$$\n",
+    "R_x(\\theta) = e^{i \\frac{\\theta}{2} X}.\n",
+    "$$\n",
+    "\n",
+    "The last section also showed us that the unitary and its corresponding Hermitian matrix have the same eigenstates. In this section, we've seen that conjugation by a unitary transforms eigenstates and leaves eigenvalues unchanged. With this in mind, it can be shown that\n",
+    "\n",
+    "$$\n",
+    "U R_x(\\theta)U^\\dagger = e^{i \\frac{\\theta}{2} ~U X U^\\dagger}.\n",
+    "$$\n",
+    "\n",
+    "By conjugating this rotation by a Clifford, we can therefore transform it to the same rotation around another axis. So even if we didn't have a direct way to perform $R_y(\\theta)$ and $R_z(\\theta)$, we could do it with $R_x(\\theta)$ combined with Clifford gates. This technique of boosting the power of non-Clifford gates by combining them with Clifford gates is one that we make great use of in quantum computing."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.3 Expanding the Gate Set <a id='expanding'></a>\n",
+    "\n",
+    "As another example of combining $R_x(\\theta)$ with Cliffords, let's conjugate it with a CNOT.\n",
+    "\n",
+    "$$\n",
+    "CX_{j,k} ~(R_x(\\theta) \\otimes 1)~ CX_{j,k} = CX_{j,k} ~ e^{i \\frac{\\theta}{2} ~ (X\\otimes 1)}~ CX_{j,k} = e^{i \\frac{\\theta}{2} ~CX_{j,k} ~ (X\\otimes 1)~ CX_{j,k}} = e^{i \\frac{\\theta}{2} ~ X\\otimes X}\n",
+    "$$\n",
+    "\n",
+    "This transforms our simple, single-qubit rotation into a much more powerful two-qubit gate. This is not just equivalent to performing the same rotation independently on both qubits. Instead, it is a gate capable of generating and manipulating entangled states.\n",
+    "\n",
+    "We needn't stop there. We can use the same trick to extend the operation to any number of qubits. All that's needed is more conjugates by the CNOT to keep copying the $X$ over to new qubits.\n",
+    "\n",
+    "Furthermore, we can use single-qubit Cliffords to transform the Pauli on different qubits. For example, in our two-qubit example we could conjugate by $S$ on the qubit on the left to turn the $X$ there into a $Y$:\n",
+    "\n",
+    "$$\n",
+    "S ~e^{i \\frac{\\theta}{2} ~ X\\otimes X}~S^\\dagger = e^{i \\frac{\\theta}{2} ~ X\\otimes Y}.\n",
```

Comment:
The equation might look clearer if it is written as
(I \otimes S) e^{i \frac{\theta}{2} X\otimes X} (I \otimes S^\dagger)  = e^{i \frac{\theta}{2}  X\otimes Y}